### PR TITLE
bin: Make S3 entry script more verbose

### LINKED ISSUE #424: Make S3 scripts safer
**Is your feature request related to a problem? Please describe.**
The [S3 scripts](https://github.com/elastisys/compliantkubernetes-apps/blob/main/scripts/S3/entry.sh) currently don't do much to ensure that you "know what you are doing". They don't explain what credentials are used, what endpoint it is trying to use or what buckets it will create/destroy. This means it is quite easy to do mistakes and increase the insecurity.

**Describe the solution you'd like**
The scripts should ask for confirmation and explain what it is about to do.

**Describe alternatives you've considered**
None

**Definition of done:**
The s3 scripts explain asks for confirmation at least before deleting things and also lists what buckets it is about to touch.

### PR DESCRIPTION
**Pull request by OlleLarsson**
**What this PR does / why we need it**:
To make the S3 scripts safer.
This just adds some more output to the entry script:
- Print buckets the will be operated on
- Print which s3cfg config is used
- Print command executed - this really contains the above information as well

**Which issue this PR fixes**:
fixes #424 

**Special notes for reviewer**:
Printing region or endpoint is not very useful imo as that would require the script to parse the s3cfg file.
I find it more sensical to really just present that "Hey, this is the s3cfg file you passed", if you need more information then a simple cat should do :)

I did not create a Yes/no promt for creating buckets as that is not really an intrusive operation.

Feedback appreciated :) 

**Example run**:
image

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [ ] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

**Review by lentzi90 — APPROVED** [review]
Excellent, thank you very much!

File: scripts/S3/entry.sh [hunk 1]
```diff
@@ -10,44 +10,57 @@ CK8S_AUTO_APPROVE=${CK8S_AUTO_APPROVE:-"false"}
...
     action="$1"
     cmd="${here}/manager.sh --$1 $buckets"
+    log_info "Using s3cmd config file: ~/.s3cfg"
 fi
 
 if [ "$action" = "delete" ] && ! ${CK8S_AUTO_APPROVE}; then
-    echo -n -e "Are you sure you want to delete all buckets? (y/n): " 1>&2
+    echo -n -e "[\e[34mck8s\e[0m] Are you sure you want to delete all buckets? (y/n): " 1>&2
     read -r reply
     if [[ "${reply}" == "n" ]]; then
```
Comment (Xartos) [inline scripts/S3/entry.sh:59]: This is not part of this change. But I would much rather have a `if [ $reply != "[yY]" ]; then exit;`
Because this script would continue if I misstyped and entered an "m" instead.

**Review by Xartos — CHANGES_REQUESTED** [review]
Great job. Saw a small thing outside of this issue that you could fix while your at it

**Review by Xartos — APPROVED** [review]

**Review by simonklb — APPROVED** [review]
The changes looks good!

If you have the time and it seems like a small change it would be nice if you could only delete the buckets for one cluster at the time. Totally optional though!

**Review by viktor-f — APPROVED** [review]
